<launch>
   <node name="ModelProcessorNode" pkg="newpackage" type="model_processor.py" output="screen"/>
</launch>